### ISSUE TITLE
on('error') is never invoked

### ISSUE THREAD

**Issue by alansouzati** [yuanqing/gulp-tape#19]
I've noticed that throw new Error does not trigger the on('Error') in gulp pipeline.

``` javascript
   gulp.src(options.testPaths)
        .pipe(gulpif(!watch, envs))
        .pipe(tape({
          reporter: tapSpec(),
          bail: true
        }))
        .on('error', (error) => {
          gutil.log(error.message);
          process.exit(1);
        })
        .pipe(gulpif(!watch, envs.reset));
```

To fix that we need to invoke callback sending the error as we do in other places in this plugin. I'm going to send a Pull Request soon

**Comment by alansouzati** [comment]
thanks for merging, would you mind to release a new version with this feature in place? 👍

**Comment by yuanqing** [comment]
Just released `0.0.9`

**Comment by alansouzati** [comment]
thanks!
